pkg/ecdsa/tss/protocol_ready.go

```diff
@@ -48,7 +51,27 @@ func readyProtocol(
 			case msg := <-readyInChan:
 				for _, memberID := range group.groupMemberIDs {
 					if msg.SenderID.Equal(memberID) {
-						readyMembers[msg.SenderID.String()] = true
+						memberAddress, err := memberIDToAddress(
+							memberID,
+							publicKeyToAddressFn,
+						)
+						if err != nil {
+							logger.Errorf(
+								"could not convert member ID to address for "+
+									"a member of keep [%v]: [%v]",
+								group.groupID,
+								err,
+							)
+							break
+						}
+						readyMembers[memberAddress] = true
+
+						logger.Infof(
+							"member [0x%v] from keep [%v] announced its readiness",
```

Comment:
For consistency, we should drop `0x` prefixing in the message as we did already in https://github.com/keep-network/keep-ecdsa/pull/654/commits/a43b3ab873000944f5b55e92f85154ac974393b9#diff-7dac81c8ac9c4245caf522776d5d4bb167a7012110db13c5a702b365b9b80f1aL64
```suggestion
							"member [%v] from keep [%s] announced its readiness",
```